portal-bridge/src/census/peer.rs

```diff
@@ -9,18 +9,18 @@ use tracing::error;
 
 #[derive(Debug, Clone)]
 pub struct LivenessCheck {
-    success: bool,
-    #[allow(dead_code)]
-    timestamp: Instant,
+    pub success: bool,
+    pub timestamp: Instant,
 }
 
-#[allow(dead_code)]
 #[derive(Debug, Clone)]
 pub struct OfferEvent {
-    success: bool,
-    timestamp: Instant,
-    content_value_size: usize,
-    duration: Duration,
+    pub success: bool,
+    pub timestamp: Instant,
+    #[allow(dead_code)]
+    pub content_value_size: usize,
+    #[allow(dead_code)]
+    pub duration: Duration,
```

Comment:
Are there plans to use these in the future?

Reply:
Yes.
Idea is to measure bandwidth and prefer peers that are faster than average.

But I didn't want to complicate initial work, and I don't think it works well with current implementation of "Weight" because peers that are working would quickly increase their weight to maximum and this wouldn't make a difference.